### PR TITLE
68 comment view kondratiuk

### PR DESCRIPTION
**Pull request by andrii-kondratiuk**

### CODE REVIEW

File: src/components/single-place/comment-view-section/comment-view-item/comment-view-item.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import './comment-view-item.scss';
+//import api from '../../../../services/review-service';
```
Comment (AndriiDruzhytskyi) [inline src/components/single-place/comment-view-section/comment-view-item/comment-view-item.js]: Please remove commented code.

File: src/components/single-place/comment-view-section/comment-view-list/comment-view-list.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import CommentViewItem from '../comment-view-item/comment-view-item';
+import './comment-view-list.scss';
+
+export default class CommentViewList extends Component {
+  render() {
+    const { commentList, placeId } = this.props;
+    console.log(placeId);
```
Comment (AndriiDruzhytskyi) [inline src/components/single-place/comment-view-section/comment-view-list/comment-view-list.js]: Please remove debug output.

File: src/components/single-place/comment-view-section/comment-view-section.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import CommentViewList from './comment-view-list/comment-view-list';
+//import CommentViewUser from './comment-view-user/comment-view-user';
+//import CommentViewRating from './comment-view-rating/comment-view-rating';
+//import CommentViewDate from './comment-view-date/comment-view-date';
+//import CommentViewBody from './comment-view-body/comment-view-body';
```
Comment (AndriiDruzhytskyi) [inline src/components/single-place/comment-view-section/comment-view-section.js]: Please remove commented code.

File: src/components/single-place/single-place.js [hunk 4]
```diff
@@ -2,47 +2,55 @@ import React, { Component } from 'react';
...
 import './single-place.scss';
+import reviewService from '../../services/review-service';
 
 export default class SinglePlace extends Component {
   state = {
     place: null
   };
 
   async componentDidMount() {
+    console.log(await reviewService.getAllComments());
```
Comment (AndriiDruzhytskyi) [inline src/components/single-place/single-place.js]: Please remove debug output.

File: src/services/review-service.js [hunk 5]
```diff
@@ -1,10 +1,23 @@
-import request from "./request";
+import request from './request';
 
-class reviewServise{
+class reviewServise {
+  async comment({ userJwt, comment, placeId, rating }) {
+    const res = await request.post('reviews', {
```
Comment (AndriiDruzhytskyi) [inline src/services/review-service.js:5]: Please use requests only in try\catch block.

**Review by AndriiDruzhytskyi — APPROVED** [review]
Good work. Ready to merge after small improvements.

**Review by Dmitry528 — APPROVED** [review]

File: src/components/single-place/comment-view-section/comment-view-item/comment-view-item.js (hunk 1 above)
Comment (andrii-kondratiuk) [reply src/components/single-place/comment-view-section/comment-view-item/comment-view-item.js]: Done

File: src/components/single-place/comment-view-section/comment-view-list/comment-view-list.js (hunk 2 above)
Comment (andrii-kondratiuk) [reply src/components/single-place/comment-view-section/comment-view-list/comment-view-list.js]: Fixed

File: src/components/single-place/comment-view-section/comment-view-section.js (hunk 3 above)
Comment (andrii-kondratiuk) [reply src/components/single-place/comment-view-section/comment-view-section.js]: Done

File: src/components/single-place/single-place.js (hunk 4 above)
Comment (andrii-kondratiuk) [reply src/components/single-place/single-place.js]: Fixed

File: src/services/review-service.js (hunk 5 above)
Comment (andrii-kondratiuk) [reply src/services/review-service.js:5]: There was no changes on my part in this code, aside from Prettier automatic code formatting.

**Review by marrybbb — APPROVED** [review]

**Pull request merged**